src/services/sync/strategies/WebUntisSyncer.js

```diff
@@ -0,0 +1,17 @@
+const Syncer = require('./Syncer');
+
+class WebUntisSyncer extends Syncer {
+	static respondsTo(target) {
+		return target === 'webuntis';
+	}
+
+	static params(params, data = {}) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['data' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4899819)